src/tools/genpolicy/src/mount_and_storage.rs

```diff
@@ -175,6 +177,63 @@ fn get_empty_dir_mount_and_storage(
     });
 }
 
+fn get_persistent_volume_claim_mount(
+    settings: &settings::Settings,
+    yaml_mount: &pod::VolumeMount,
+    yaml_volume: &volume::Volume,
+    p_mounts: &mut Vec<policy::KataMount>,
+    storages: &mut Vec<agent::Storage>,
+    persistent_volume_claims: &Vec<PersistentVolumeClaim>,
+    propagation: &str,
+    access: &str,
+) {
+    let volume_pvc = yaml_volume.persistentVolumeClaim.as_ref().unwrap();
+    let is_blk_mount =
+        persistent_volume_claims.iter()
+        .find(|pvcResource| pvcResource.metadata.name.as_ref() == Some(&volume_pvc.claimName))
```

Comment:
I think we should log a warning here if we can't find the PVC: either it wasn't provided via `-pvc` or it's not included in the spec file, but in either case we would not be able to determine if this is a block mount.

Reply:
Warning sounds good - and perhaps return an Error value if we don't already.

Reply:
Done as a warning. Didn't error out as that would break existing users.